kroxylicious-filters/kroxylicious-record-validation/src/test/java/io/kroxylicious/proxy/filter/validation/validators/bytebuf/JsonSchemaBytebufValidatorTest.java

```diff
@@ -78,18 +79,24 @@ public static void initMockRegistry() {
         registryServer.start();
 
         registryServer.stubFor(
-                get(urlEqualTo("/apis/registry/v2/ids/globalIds/1?dereference=false"))
+                get(urlEqualTo("/apis/registry/v3/ids/contentIds/1"))
                         .willReturn(WireMock.aResponse()
                                 .withHeader("Content-Type", "application/json")
                                 .withBody(JSON_SCHEMA)));
 
         registryServer.stubFor(
-                get(urlEqualTo("/apis/registry/v2/ids/globalIds/1/references"))
+                get(urlEqualTo("/apis/registry/v3/ids/globalIds/1?dereference=false"))
```

Comment:
Are you sure you still need the dereference=false stubbing?  I tried removing out and the tests continued to pass.